src/core/main.cpp

```diff
@@ -137,6 +142,14 @@ int main( int argc, char * * argv )
 		}
 	}
 
+#ifndef LMMS_WIN32
```

Comment:
Oh... I see what you are saying... I wonder if there's a way to edit the YML to only run the script for LMMS/lmms so that the PRs can be flagged as pass/fail without bugging out the forks.